notebooks/heading_similarity.ipynb

```diff
@@ -0,0 +1,1668 @@
+{
```

Comment:
<p>How is this different from the code above? It looks like the plot is more or less the same (at least as far as I can tell).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/indices/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Weird, it looks like the cells got duplicated. I'll go remove the extras</p>